mentat/terminal/client.py

```diff
@@ -252,3 +258,39 @@ def run_cli():
         auto_tokens,
     )
     terminal_client.run()
+
+
+# This needs to stay in the terminal client since it's done on start and uses print/exit,
+# which is why we have an almost identical version in include_files
```

Comment:
Similar to [this comment I left](https://github.com/AbanteAI/mentat/pull/143/files#r1355908637), by copying over the queue messages to new subscriber queue's we can send "interrupt" messages before the Terminal Client is subscribed to the Session and have the shutdown work properly.

Reply:
This is a bit different (and won't be the same for every client); originally, the only reason we did this here is because zsh and some other shells automatically glob expand when you run commands, so by having this done at the very start we guaranteed no differences between different shells. In this PR I actually added glob expand to the get include files, so this function isn't actually necessary. Thinking about it more though, I don't think we need this function at all; if there are extremely small differences between shells because of it (which I don't even think there will be), I don't think it matters. I'll probably end up removing this function (as long as you agree)